apps/browser-extension-wallet/src/stores/slices/activity-detail-slice.ts

```diff
@@ -222,7 +222,12 @@ const buildGetActivityDetail =
       includedUtcTime: blocks?.utcTime,
       // TODO: store the raw data here and transform it later so we always have the raw data when needed.(LW-9570)
       votingProcedures: votingProceduresTransformer(tx.body.votingProcedures),
-      proposalProcedures: governanceProposalsTransformer(cardanoCoin, tx.body.proposalProcedures),
+      proposalProcedures: governanceProposalsTransformer(
+        cardanoCoin,
+        coinPrices,
+        fiatCurrency,
+        tx.body.proposalProcedures
+      ),
```

Comment:
May not be very relevant here, but in general I'd suggest introducing a single `object` param when you have more than ~3 params. This gives you also order flexibility and you don't need to bother with optional properties.

```suggestion
      proposalProcedures: governanceProposalsTransformer({
        cardanoCoin,
        coinPrices,
        fiatCurrency,
        proposalProcedures: tx.body.proposalProcedures
      }),
```